ext/standard/basic_functions.stub.php

```diff
@@ -493,6 +493,8 @@ function ini_alter(string $option, string|int|float|bool|null $value): string|fa
 
 function ini_restore(string $option): void {}
 
+function ini_bytes(string $option): int {}
```

Comment:
IMO name of the parameter should be something like `$value` instead of `$option` as according to your examples in the PR description what you pass here is a string representation of a value.

Reply:
Thanks for the good catch. That was an artifact of starting by trying to return the internal value for a php.ini directive before I gave up and realized that was too broad of a scope.

Renamed to `$shorthand` in 7f626f1049